crates/router/src/connector/stripe/transformers.rs

```diff
@@ -1453,6 +1483,11 @@ pub struct StripeRedirectToUrlResponse {
     url: Url,
 }
 
+#[derive(Clone, Debug, Eq, PartialEq, Deserialize, Serialize)]
+pub struct StripeRedirectToQR {
```

Comment:
Nit, optional: naming convention 
```suggestion
pub struct StripeRedirectToQr {
```

Line 1443 would also have to be updated if you make this change.